src/main/java/seedu/kitchenhelper/command/ListIngredientCommand.java

```diff
@@ -64,9 +65,9 @@ public String listIngredients(String category, ArrayList<Ingredient> ingredientL
                 for (int i = 0; i < ingredientList.size(); i++) {
                     Ingredient ingredientObj = ingredientList.get(i);
                     if (ingredientObj.getCategoryName().equalsIgnoreCase(category)) {
-                        result += ingredientObj.getIngredientName() + "|"
-                                + Integer.toString(ingredientObj.getQuantity()) + "|"
-                                + Double.toString(ingredientObj.getPrice()) + "|"
+                        result += "[" + (i + 1) + "] Ingredient Name : " + ingredientObj.getIngredientName() + " | "
```

Comment:
![image](https://user-images.githubusercontent.com/32284166/78216927-870bf380-74ed-11ea-87c2-b478beaff142.png)

As seen from the image above, would it be better to remove the `Ingredient Name:` string before the actual ingredient name? 😆 It might be more user friendly, but just a suggestion!